include/zephyr/bluetooth/gatt.h

```diff
@@ -1001,6 +1032,11 @@ struct _bt_gatt_ccc {
  *  @return number of bytes read in case of success or negative values in
  *          case of error.
  */
+/** @cond INTERNAL_HIDDEN
+ *  @note Only use this with attributes which user_data is a _bt_gatt_ccc.
+ *  _bt_gatt_ccc being the internal representation of CCC value.
+ */
+ /** @endcond */
```

Comment:
I wonder if 
```suggestion
/** @cond INTERNAL_HIDDEN
 *  @note Only use this with attributes which user_data is a _bt_gatt_ccc.
 *  _bt_gatt_ccc being the internal representation of CCC value.
 *
 *  @endcond 
 */
```

works as well

Reply:
Won't, because it complaints if `@endcond` is complaining if it's within a comment block :) (plus other places in Zephyr also follow this extra comment block for endcond.)